cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -71,6 +71,15 @@ data:
 {{- else }}
         listen {{ $nginxPort }};
 {{- end }}
+        location ~* \.(js|html|css|png|jpg|jpeg|gif)$ {
+            add_header Cache-Control "max-age=43200";
```

Comment:
This is 30 days correct? Seems fine, but could also probably go longer...

Reply:
This is actually 12h. We can increase it though.